README.md

```diff
@@ -0,0 +1,51 @@
+# Abobot
+
+## Description
+
+This project is a music-playing bot written in Python, mainly using the youtube-dl and yt-dlp libraries. 
+It allows you to play music from various sources, with full support currently only for YouTube. 
+To use the bot, you need to create a .env file at the root of the project to specify the bot token. 
+An example of usage is described in the .env_example file. For convenient program launching, Docker can be used with the make command. 
```

Comment:
Should be more like: "List of required environment variables is provided in .env_example file"